gui/create-item.lua

```diff
@@ -385,14 +391,20 @@ function createCorpsePiece(creator, bodypart, partlayer, creatureID, casteID, ge
        -- search through the target creature's body parts and bring back every one which has the desired material
        if creatorRaceRaw.tissue[creatorBody.body_parts[i].layers[n].tissue_id].tissue_material_str[1] == layerMat and creatorBody.body_parts[i].token ~= "SKULL" and not creatorBody.body_parts[i].flags.SMALL then
         item.body.components.body_part_status[i].missing = false
-        item.body.components.body_part_status[i].grime = 2
```

Comment:
was this removal intentional? setting `grime` to 2 looks strange, but why was it here to begin with?

Reply:
it was. this was a leftover from me experimenting with what made corpsepieces work.